.github/workflows/_shared-first-time-setup.yml

```
@@ -0,0 +1,64 @@
+name: "Repository First Time Setup automation (shared)"
+
+on:
+  workflow_call:
+    inputs:
+        esf_version:
+          type: string
+          default: '7.2.2'
+          required: true
+        archetype_url:
+          type: string
+          default: 'https://changeme'
+          required: true
+        groupId:
+          type: string
+          default: ''
+          required: true
+        artifactId:
+          type: string
+          default: ''
+          required: true
+
+jobs:
+  init:
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Download ESF archetype
+      uses: carlosperate/download-file-action@v1
+      with:
+        file-url: ${{ github.event.inputs.archetype_url }}
+      if: startsWith(${{ github.event.inputs.archetype_url }}, 'https://eth-repo.s3.amazonaws.com/esf')
```

The `if` statement is ignored, try moving it after the `name`

```suggestion
    - name: Download ESF archetype
      if: startsWith(${{ github.event.inputs.archetype_url }}, 'https://eth-repo.s3.amazonaws.com/esf')
      uses: carlosperate/download-file-action@v1
      with:
        file-url: ${{ github.event.inputs.archetype_url }}
```